libs/feature/catalog/src/lib/source-label/source-label.component.ts

```diff
@@ -12,15 +8,9 @@ import { SourcesService } from '../sources/sources.service'
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class SourceLabelComponent {
-  sourceLabel$: Observable<CatalogSource>
   @Input() set catalogUuid(uuid: string) {
-    this.sourceLabel$ = this.sourcesService
-      .getSource(uuid)
-      .pipe(map((source) => source.label[this.langService.iso3]))
+    this.sourcesService.setSourceUuid(uuid)
```

Comment:
I don't like this sorry, cause if the component is not in the page, then the source service won't update its source.
2 options:
- we keep to store the current source uuid in the `sourceService`, wich might not be conceptually ideal, sorry i was not very aware and clear in my last review. If we do this, we should update the source uuid on record-state metadata change, not within this component.
- we don't store any uuid in the service, but the `getSourceLabel(uui: string) `method is in the source Service, not in the component. I think it's better for now.